docs/source/installation.rst

```diff
@@ -2,6 +2,37 @@
 Installation
 ========================
 
+Start using MONAI Label locally and run installlation with your familiar visualization tools. Stable version software represents the currently tested
+and supported visualization tools with latest release of MONAI Label. Weekly preview version is available if users want the latest feature, not fully tested.
+
+
+Current Milestone Release of MONAI Label:
+
+* `0.4.2 <https://pypi.org/project/monailabel/>`_
+
+MONAI Label Supported Stable Visualization Tools:
+
+* 3D Slicer: `Version 5.0.3 <https://download.slicer.org/>`_
```

Comment:
lets not add the version here.. client normally have their own different cycle.. let it be linked to downloads page only..

even for monailabel.. otherwise for every release we have to make sure we update this section as well

Reply:
Hi @SachidanandAlle ,

Agree, but I think we should say the supported versions here.
I encountered wrong version issue of 3D Slicer twice.. And our SA also reported similar issue before: users don't know which versions of 3D Slicer can be compatible.

Thanks.

Reply:
Actually, I would suggest to design some unit tests or integration tests for the supported versions of 3D Slicer.
For example, we have the unit tests for MONAI core with different python versions:
https://github.com/Project-MONAI/MONAI/blob/dev/.github/workflows/pythonapp-min.yml#L77

Thanks.

Reply:
We can mention version >= if that helps our new users or SA

Reply:
@SachidanandAlle , but I faced that before that the latest 3D Slicer can't work with MONAI Label..

Reply:
3D slicer integration test is possible through slicer non gui mode.. great if one can get someone's help to explore and add those tests 🙂

Reply:
MONAILabel was not available for Slicer-4.11.20210226 which required using the latest Slicer preview build. However Slicer 5.0 was released recently so there shouldn’t be issues moving forward as MonaiLabel should be supported on both the stable and latest preview. The latest general improvements to 3D Slicer will always be available in the latest Slicer Preview if a workflow improvement is made to related code like Segmentation capabilities.

Reply:
Yes..  r we building latest plugin (daily) for both.. or its just for preview.. in that case may be better to add this info for users as well.. to choose preview vs stable...

Reply:
The latest version of the plugin is built daily for the latest stable and for the latest preview build. See https://discourse.slicer.org/t/new-feature-automatic-update-of-extensions/24416 for details about updating an extension for the latest Stable. This is the easiest method.

Extensions can’t usually be updated for a Slicer Preview build because an increment in the Slicer revision number will mean you have to download the new Slicer preview to get the new MonaiLabel version for that specific Slicer Preview build. You could utilize the “bookmark” functionality to easily install MonaiLabel again once you installed a new version of Slicer Preview. However it’s not an “update” action in this case but a new install for a new version.

Reply:
As 3D Slicer version >= 5.0 is tested. We can mention the version range > 5.0 for stable and preview version in this installation page, in case not to confuse new users or someone with old version 3D Slicer.

Do we also have version issues regarding OHIF or QuPath? OHIF is a web-based tools, should be fine if version is not provided. 
```
* 3D Slicer: `Stable and Preview Version >= 5.0 <https://download.slicer.org/>`_
* OHIF: `Version >= 3.1 <https://ohif.org/>`_
* QuPath: `Version >= 0.3.2 <https://qupath.github.io/>`_
```